### PR TITLE
Fix import of ModelSystems

### PR DESCRIPTION
**Pull request by SRFU-NN**
Moved `ModelSystems` to a system where they are created on demand. This should fix the other issue mentioned in #262 , while also making importing ProcessOptimizer faster

Also discovered a bug while looking at the code.

If noise size is set implicitly and the noise is independent of the score (ConstantNoise), the noise size should be set to 1% of the score span, but the parameter that was set was the wrong one.

I don't think it affects any of our model systems, since they all have the noise size set explicitly.

### CODE REVIEW

File: examples/Model systems.ipynb
Comment (sqbl) [inline examples/Model systems.ipynb:1]: Seems to work now. I suggest an edit to either the braninHoo model or to the notebook. In the notebook, under "Changing the noise type", it says that the noise is constant, while in fact Branon-hoo defaults to proportional noise. It should be changed in one of the two placed to be aligned.

Comment (SRFU-NN) [reply examples/Model systems.ipynb:1]: Good catch, fixed the description. Also deleted the last cell, it wasn't needed.

**Comment by SRFU-NN** [comment]
@dk-teknologisk-mon : This is rather a big conceptual change, so I would like your input.

Also, Bertelsen discovered that there was an inconsistency with the noise type for the Branin-Hoo model system. I have made the default noise proportional to the signal, so there is little noise around the optima, and a lot elsewhere. Let's discuss whether this is what we want on the nerdy meeting next week.

**Comment by SRFU-NN** [comment]
Now fixes #268

**Review by sqbl — APPROVED** [review]
LGTM

**Pull request merged**